Makefile

```diff
@@ -70,7 +70,7 @@ lint-diff:
 
 lint:
 	# stop the build if there are Python syntax errors or undefined names
-	$(PYTHON) -m flake8 . --count --exclude=$(FLAKE_EXCLUDE) --select=E9,F63,F7,F82 --show-source --statistics
+	$(PYTHON) -m flake8 . --count --exclude=$(FLAKE_EXCLUDE) --ignore=E9,F63,F7,F82 --show-source --statistics
```

Comment:
??? We should ignore only those tests that we still fail.  These are tests that we should definitely should _not_ fail.

Reply:
Oops, sorry. Completely misread the title.

Reply:
I will revert the last commit but the failure it in the `lint-diff` step:
```
58    E203 whitespace before ':'
2     E265 block comment should start with '# '
1     E402 module level import not at top of file
150   E501 line too long (98 > 88 characters)
3     E741 ambiguous variable name 'l'
2     F405 'translate' may be undefined, or defined from star imports: openlibrary.catalog.marc.fast_parse
4     F601 dictionary key 'resource_type' repeated with different values
5     F841 local variable 'cat' is assigned to but never used
273   W503 line break before binary operator
17    W605 invalid escape sequence '\d'
```
W503 should be ignored: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-breaks-binary-operators

E203 should be ignored as well: https://black.readthedocs.io/en/stable/the_black_code_style.html#slices

Should we increase the line length to 100?

The rest should probably be fixed manually.

Reply:
Increase `--max-line-length=` until the E501 errors go away and then we can create a __future__ issue to wrap lines to 88.